_posts/2020-07-29-community-update.md

```diff
@@ -0,0 +1,208 @@
+
+Also, events are starting to pick up again, so we've put together a list of some great ones you can (virtually) attend in August!
+
+{% toc %}
+
+# The Past Month in Flink
+
+## Flink Releases
+
+### Flink 1.11
+
+A couple of weeks ago, Flink 1.11 was announced in what was (again) the biggest Flink release to date (_see ["A Look Into the Evolution of Flink Releases"](#a-look-into-the-evolution-of-flink-releases)_)! The new release brought significant improvements to usability as well as new features to Flink users across the API stack. Some highlights of Flink 1.11 are:
+
+  * Unaligned checkpoints to cope with high backpressure scenarios;
+
+  * The new source API, that simplifies and unifies the implementation of (custom) sources;
+
+  * Support for Change Data Capture (CDC) and other common use cases in the Table API/SQL;
+
+  * Pandas UDFs and other performance optimizations in PyFlink, making it more powerful for data science and ML workloads.
+
+For a more detailed look into the release, you can recap the [announcement blogpost](https://flink.apache.org/news/2020/07/06/release-1.11.0.html) and join the upcoming meetup on [“What’s new in Flink 1.11?”](https://www.meetup.com/seattle-flink/events/271922632/), where you’ll be able to ask anything release-related to Aljoscha Krettek (Flink PMC Member). The community has also been working on a series of blogposts that deep-dive into the most significant features and improvements in 1.11, so keep an eye on the [Flink blog](https://flink.apache.org/blog/)!
+
+### Flink 1.11.1
+
+Shortly after releasing Flink 1.11, the community announced the first patch version to cover some outstanding issues in the major release. This version is **particularly important for users of the Table API/SQL**, as it addresses known limitations that affect the usability of new features like changelog sources and support for JDBC catalogs. 
+
+You can find a detailed list with all the improvements and bugfixes that went into Flink 1.11.1 in the [announcement blogpost](https://flink.apache.org/news/2020/07/21/release-1.11.1.html).
+
+<hr>
+
+## Gearing up for Flink 1.12
+
+The Flink 1.12 release cycle has been kicked-off last week and a discussion about what features will go into the upcoming release is underway in [this @dev Mailing List thread](https://lists.apache.org/thread.html/rb01160c7c9c26304a7665f9a252d4ed1583173620df307015c095fcf%40%3Cdev.flink.apache.org%3E). While we wait for more of these ideas to turn into proposals and JIRA issues, here are some recent FLIPs that are already being discussed in the Flink community:
+
+<table class="table table-bordered">
+  <thead>
+    <tr>
+      <th>FLIP</th>
+      <th></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><a href="https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=158866298">130</a></td>
```

Comment:
```suggestion
      <td><a href="https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=158866298">FLIP-130</a></td>
```

Reply:
Dropped the "FLIP" because of Jekyll-related stuff (i.e. makes the table look ugly 😅 ), but it does look weird without it.